.cloudbuild/scripts/internal/changes/changes.go

```diff
@@ -31,8 +31,9 @@ import (
 
 // Changes describes the kind of changes found in the analysed workspace.
 type Changes struct {
-	Docs bool
-	Code bool
+	Docs       bool
+	Code       bool
+	Enterprise bool
```

Comment:
We have more than one submodule. Is it worth making this generic over any submodule change, or should we stick just with changes to `./e` for now?